spec/index.bs

```diff
@@ -10191,23 +10192,31 @@ and {{GPUTextureUsage/STORAGE_BINDING|GPUTextureUsage.STORAGE_BINDING}} usage re
             <th>Format
             <th>{{GPUTextureSampleType}}
             <th>{{GPUTextureUsage/RENDER_ATTACHMENT}}
+            <th>multisampling
+            <th>resolve
             <th>{{GPUTextureUsage/STORAGE_BINDING}}
     </thead>
     <tr><th class=stickyheader>8-bit per component<th><th><th>
     <tr>
         <td>{{GPUTextureFormat/r8unorm}}
         <td>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
         <td>&checkmark;
+        <td>&checkmark;
+        <td>&checkmark;
         <td>
     <tr>
         <td>{{GPUTextureFormat/r8snorm}}
         <td>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
         <td>
         <td>
```

Comment:
AFAICT multisampling should always be valid for r8snorm in Metal, D3D FL11_0, and Vulkan (Vulkan doesn't require r8snorm to have RENDER_ATTACHMENT but it does require it to have SAMPLED and I think SAMPLED requires multisampling (sampledImageColorSampleCounts).

Reply:
I just skipped multisample flag for textures that can't be rendered to. How are you going to populate their contents if not by rendering?

I also thought about just validating this at texture creation, i.e. if it's multisampled then require RENDER_ATTACHMENT usage, but then decided against it, since technically one should be able to create 2 textures, render to one and copy from one to another. So the other would not have RENDER_ATTACHMENT.

Reply:
> I just skipped multisample flag for textures that can't be rendered to. How are you going to populate their contents if not by rendering?

Ah I see. It does seem a bit weird not to have a checkmark just because the format doesn't allow RENDER_ATTACHMENT or STORAGE_BINDING. When we inevitably add support for more renderable and storable formats we'll just have to fix this table. (Maybe we can leave a `<!-- Allow this when this format is renderable or storable -->` in lieu of a `<!-- Vulkan -->` type comment)

Reply:
Ok, whether or not this makes sense to allow we can discuss separately. I added the checkboxes for multisampling where appropriate here.

Reply:
#2465

Reply:
@Jiawei-Shao discovered that Vulkan does not require VK_FORMAT_R8_SNORM, VK_FORMAT_R8G8_SNORM and VK_FORMAT_R8G8B8A8_SNORM to support VK_SAMPLE_COUNT_4_BIT.

At least SwiftShader doesn't support it, but I don't know about other drivers. Could we check how many devices we lose by requiring these formats to support sampleCount=4?